src/pages/ThorChainLP/components/LpType.tsx

```diff
@@ -120,21 +130,43 @@ export const LpType = ({ assetId, opportunityId, onAsymSideChange }: DepositType
       const radio = getRadioProps({ value: option.value })
       const optionAssetIds = makeAssetIdsOption(option.value as AsymSide | 'sym')
 
+      const isDisabled = (() => {
+        if (!isWithdraw) return false
+
+        if (isSymPositionType) return false
+        if (isRunePositionType) return option.value !== AsymSide.Rune
+        if (isAssetPositionType) return option.value !== AsymSide.Asset
+
+        return false
+      })()
+
       return (
-        <TypeRadio key={`type-${index}`} {...radio}>
-          <PoolIcon assetIds={optionAssetIds} size='xs' />
-          <Flex mt={4} fontSize='sm' justifyContent='space-between' alignItems='center'>
-            <TypeLabel assetIds={optionAssetIds} />
-            {optionAssetIds.length === 1 && (
-              <Box as='span' color='text.subtlest' fontSize='md' className='asym-icon'>
-                <BiSolidBoltCircle />
-              </Box>
-            )}
-          </Flex>
+        <TypeRadio key={`type-${index}`} {...radio} isDisabled={isDisabled}>
+          <Tooltip isDisabled={!isDisabled} label={translate('pools.withdrawTypeNotAvailable')}>
```

Comment:
nitpick - we could be explicit about deposit state as well in case `isDisabled` logic changes in the future for deposit so we don't show a withdrawal tooltip for deposit
```suggestion
          <Tooltip isDisabled={!isDisabled || isDeposit} label={translate('pools.withdrawTypeNotAvailable')}>
```

Reply:
Changed in [b602e31](https://github.com/shapeshift/web/pull/7351/commits/b602e31990567afc87ae66b6a0c081c5b9bbdb26)